modules/multiprocess_logging/multiprocess_logging.py

```diff
@@ -0,0 +1,18 @@
+import inspect
+import logging
+
+DEBUG = logging.DEBUG
+INFO = logging.INFO
+WARNING = logging.WARNING
+ERROR = logging.ERROR
+CRITICAL = logging.CRITICAL
+
+def message_and_metadata(message, frame):
+        function_name = frame.f_code.co_name
+        filename = frame.f_code.co_filename
+        line_number = inspect.getframeinfo(frame).lineno
+
+        return f'[{filename} | {function_name} | {line_number}] {message}'
```

Comment:
the frame object cannot be pickled so you can't put it in a queue, this is why we need this helper function, since we probably don't want to write this every time we want to log